frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelineRuns.cy.ts

```diff
@@ -493,6 +501,178 @@ describe('Pipeline runs', () => {
+            .within(() => pipelineRunFilterBar.findExperimentInput().type('Test Experiment 1'));
+
+          // Mock runs (filtered by selected experiment)
+          archivedRunsTable.mockGetArchivedRuns(
+            mockArchivedRuns.filter((mockRun) => mockRun.experiment_id === 'test-experiment-1'),
+            1,
+          );
+
+          // Select an experiment to filter by
+          pipelineRunFilterBar.selectExperimentByName('Test Experiment 1');
+
+          // Verify only rows with selected experiment exist
+          archivedRunsTable.findRows().should('have.length', 2);
+          archivedRunsTable.getRowByName('Test archived run 1').find();
+          archivedRunsTable.getRowByName('Test archived run 2').find();
+        });
+
+        it('filter by pipeline version', () => {
+          // Verify initial run rows exist
+          archivedRunsTable.findRows().should('have.length', 2);
+
+          // Select the "Pipeline version" filter, select a value to filter by
+          pipelineRunsGlobal
+            .findArchivedRunsToolbar()
+            .within(() => pipelineRunsGlobal.selectFilterByName('Pipeline version'));
+
+          // Mock runs (filtered by selected version)
+          archivedRunsTable.mockGetArchivedRuns(
+            mockArchivedRuns.filter(
+              (mockRun) =>
+                mockRun.pipeline_version_reference.pipeline_version_id === 'test-version-1',
+            ),
+            1,
+          );
+
+          // Select version to filter by
+          pipelineRunFilterBar.selectPipelineVersionByName('Test Version 1');
+
+          // Verify only rows with selected experiment exist
+          archivedRunsTable.findRows().should('have.length', 1);
+          archivedRunsTable.getRowByName('Test archived run 1').find();
+        });
+
+        // TODO: .clear() command causing full table to clear, not just date field
+        // it('filter by started', () => {
+        //   // Verify initial run rows exist
+        //   archivedRunsTable.findRows().should('have.length', 2);
+
+        //   // Select the "Started" filter, select a value to filter by
+        //   pipelineRunsGlobal
+        //     .findArchivedRunsToolbar()
+        //     .within(() => pipelineRunsGlobal.selectFilterByName('Started'));
+
+        //   // Mock runs (filtered by start date), type a start date
+        //   archivedRunsTable.mockGetArchivedRuns(
+        //     mockArchivedRuns.filter((mockRun) => mockRun.created_at.includes('2024-02-05')),
+        //     1,
+        //   );
+        //   pipelineRunsGlobal
+        //     .findArchivedRunsToolbar()
+        //     .within(() => pipelineRunFilterBar.findStartDateInput().type('2024-02-05'));
+
+        //   // Verify only rows with selected start date exist
+        //   archivedRunsTable.findRows().should('have.length', 1);
+        //   archivedRunsTable.getRowByName('Test archived run 1').find();
+
+        //   // Mock runs with a cleared filter before updating again
```

Comment:
maybe you can uncomment these lines of code

Reply:
This section is currently causing a bug where everything is cleared, not just the date field. @christianvogt suggested I leave it commented out for now

Reply:
I vaguely recall what we was happening from last week. But if we have a bug, please log a jira issue to get fixed. This code can reference the bug, and similarly have the jira reference this test so it can be uncommented and updated at the same time.